torch/_inductor/config.py

```diff
@@ -48,6 +48,9 @@
 # Optimize away split cat patterns (Experimental)
 split_cat_fx_passes = True
 
+# enable pattern match with group fusion (using fbgemm)
+group_fusion = False
```

Comment:
The major reason of making it false is: fbgemm group linear uses a different linear algebra compared to PT, so it's not numerical on par with PT linear, differing by the 4th digit right now.

Reply:
Unfortunately it's worse compared to fp64. I'm testing with [```same```](https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/utils.py#L911) function, and this is the output:
```
[2023-07-13 16:16:05,949] torch._dynamo.utils: [ERROR] RMSE (res-fp64): 0.00017, (ref-fp64): 0.00000 and shape=torch.Size([4, 16])
[2023-07-13 16:16:06,108] torch._dynamo.utils: [ERROR] RMSE (res-fp64): 0.00004, (ref-fp64): 0.00000 and shape=torch.Size([4, 6])
```

> Can you do a full CI run with this enabled?

The unit tests are running with ```group_fusion ``` enabled, but I think OSS CI doesn't have ```fbgemm```, so they are skipped. In fbcode, I added ```fbgemm``` as inductor unit tests dependency, so they are tested. Do you think we should set up ```fbgemm``` for OSS CI as well?

Reply:
We should open source the grouped gemm code.